packages/styling/src/interfaces/ISerifs.ts

```diff
@@ -0,0 +1,21 @@
+import { IRawStyle } from '@uifabric/merge-styles';
+
+/**
+ * @internal
+ * Experimental interface for decorative styling in a theme.
+ */
+export interface ISerifs {
```

Comment:
They say the single hardest thing in computer science is naming things and off by one errors.

I'm not thrilled with the name here, but it gets the point across. Extra, decorative styles that don't strongly influence functionality but help things feel better. "Flair" doesn't sound quite right, and "styles" or "visualizations" seem to vague.

Reply:
Doesn't Serif almost always have to do with font? This seems a bit misleading.

Reply:
I'm also confused by what a 'serif' is in this context. I would have assumed it was something font-related.

Reply:
Agreed. Naming things is tough, but serif isn't right here, especially since themes also support fonts. 

What about `IDecorations`?

Reply:
Are these `elevations`?

Reply:
@dzearing -- I think these are meant to be other kinds of customizations that aren't fonts or colors.

Reply:
Yes the single most confusing PR name I've ever seen. @phkuo you know you can ask a designer for help! May I suggest Surface styling?

Reply:
@Jahnp I'm unclear then on the use case. Can you or @phkuo provide an example of the use case?

Reply:
@dzearing For theming shadows and rounded corners. For example, if you want to use a glow instead of a box-shadow, or if you want sharp corners instead of rounded corners. For now, we would set the default roundedCorner value to 0, and in the test Fluent theme, set it to 2.